mem0/memory/utils.py

```
@@ -1,3 +1,5 @@
+import json
```

Ruff (0.7.0)

`json` imported but unused

Remove unused import: `json`

(F401)